{"variants":[{"paths":["\/documentation\/mobilepaymentssdkapi\/readerconnectionfailurereason\/revokedbydevice"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"symbol","schemaVersion":{"patch":0,"major":0,"minor":3},"metadata":{"modules":[{"name":"MobilePaymentsSDK"}],"externalID":"c:@M@MobilePaymentsSDKAPI@E@SQMPReaderConnectionFailureReason@SQMPReaderConnectionFailureReasonRevokedByDevice","symbolKind":"case","title":"ReaderConnectionFailureReason.revokedByDevice","fragments":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"text":"revokedByDevice","kind":"identifier"}],"roleHeading":"Case","role":"symbol"},"identifier":{"url":"doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI\/ReaderConnectionFailureReason\/revokedByDevice","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI","doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI\/ReaderConnectionFailureReason"]]},"sections":[],"abstract":[{"text":"An error with the device occurred while attempting to establish a connection to Square.","type":"text"}],"primaryContentSections":[{"declarations":[{"languages":["swift"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"revokedByDevice"}],"platforms":["iOS"]}],"kind":"declarations"}],"references":{"doc://com.squareup.MobilePaymentsSDKAPI/documentation/MobilePaymentsSDKAPI":{"url":"\/documentation\/mobilepaymentssdkapi","abstract":[],"identifier":"doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI","title":"MobilePaymentsSDK","type":"topic","role":"collection","kind":"symbol"},"doc://com.squareup.MobilePaymentsSDKAPI/documentation/MobilePaymentsSDKAPI/ReaderConnectionFailureReason":{"role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"ReaderConnectionFailureReason","kind":"identifier"}],"url":"\/documentation\/mobilepaymentssdkapi\/readerconnectionfailurereason","title":"ReaderConnectionFailureReason","identifier":"doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI\/ReaderConnectionFailureReason","abstract":[{"type":"text","text":"Indicates why a reader might have failed to connect to Square."}],"kind":"symbol","navigatorTitle":[{"text":"ReaderConnectionFailureReason","kind":"identifier"}],"type":"topic"},"doc://com.squareup.MobilePaymentsSDKAPI/documentation/MobilePaymentsSDKAPI/ReaderConnectionFailureReason/revokedByDevice":{"role":"symbol","title":"ReaderConnectionFailureReason.revokedByDevice","identifier":"doc:\/\/com.squareup.MobilePaymentsSDKAPI\/documentation\/MobilePaymentsSDKAPI\/ReaderConnectionFailureReason\/revokedByDevice","kind":"symbol","type":"topic","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"revokedByDevice","kind":"identifier"}],"url":"\/documentation\/mobilepaymentssdkapi\/readerconnectionfailurereason\/revokedbydevice","abstract":[{"type":"text","text":"An error with the device occurred while attempting to establish a connection to Square."}]}}}